docs/AppendixA-FAQ.md

```diff
@@ -1 +1,47 @@
+|`Number of MINC files that will be considered for inserting into the database: 0. No data could be converted into valid MINC files. Localizers will not be considered!`|The upload contains only ‘localizer’ type SeriesDescription|Localizers are not processed by default in Loris|
+|My upload DICOM has 6 modalities but only 5 are inserted; the 6th one has no MINC generated and nothing in the LORIS MRI Violated Scans module. In addition, during insertion, the message `Number of MINC files that will be considered for inserting into the database: 5` is displayed, confirming that only 5 files are considered|Probably the scan is that of a `localizer` acquisition. `tarchiveLoader` is specifically 'instructed' to exclude this modality|No action needed. This is the expected behavior of the LORIS-MRI insertion pipeline|
+|My upload DICOM has 6 modalities but only 5 are inserted; the 6th one has no MINC generated and nothing in the LORIS MRI Violated Scans module. In addition, during insertion, the message `Number of MINC files that will be considered for inserting into the database: 5` is displayed, confirming that only 5 files are considered|Probably the DICOM headers have blank SeriesNumber. LORIS-MRI generates MINC files and names them temporarily (in intermediate insertion steps) based on their SeriesNumber, before it proceeds to renaming them once the protocol is identified from the `mri_protocol` table. Having no SeriesNumber defaults to `0`, causing overwrites on the intermediate MINC files generated|Ensure that your DICOM headers include a non-blank SeriesNumber for every acquisition in that specific study|
+|`The target directory does not contain a single DICOM file`|Probably the DICOM headers have blank StudyUID. The logic of insertion within LORIS-MRI depends on a StudyUID header|Ensure that your DICOM headers include a non-blank StudyUID header|
+|My resting-state fMRI scans are tagged as task fMRI although I have 2 entries in the `mri_protocol` table|The resting-state scan has parameters that match those of the task entry of the `mri_protocol` table, and the task-related entry in the `mri_protocol` table precedes that of the resting-state fMRI|Make sure the `mri_protocol` table has parameters that discern between all the study acquired modalities in an **exclusive** manner (i.e. no two row entries have overlapping parameters across all their columns)|
+|`no MINCs inserted`|Possibly all the MINC images are violated scans|Check the details of the image headers (in Imaging Browser or using `mincheader` against the `mri_protocol` table entries, and adjust the table protocol parameters accordingly|
```

Comment:
Technically, if no MINC are inserted, you should not be able to see them via the Imaging Browser module, no?

Maybe in the 3rd column, you could say: "Check the details of the image headers (in the MRI Violated Scans module or using `mincheader` against the `mri_protocol` table entries) and adjust the table protocol parameters accordingly.

PS: was missing the closing ")"

Reply:
of course they won't be in imaging browser. Thanks for the catch.